src/amber/environment/loader.cr

```diff
@@ -2,11 +2,14 @@ module Amber::Environment
   class Loader
     def initialize(@environment : Amber::Environment::EnvType = Amber.env.to_s,
                    @path : String = Amber.environemnt_path)
-      raise Exceptions::Environment.new(@path, @environment) unless settings_file_exist?
     end
 
     def settings
-      Settings.from_yaml(settings_content.to_s)
+      if content = settings_content
```

Comment:
Do we really want to load default values if they don't have env files?

Reply:
We have to load default because the CLI will not be able to run `amber new {project_name}`

Reply:
That is a limitation added by the CLI logging PR. It doesn't really seem like the CLI needs a configurable logger though. Is this the best way to go about it?